deploy/Ubuntu1804/ntnx-csi-provisioner.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       - name: csi-provisioner
         image: quay.io/k8scsi/csi-provisioner:v1.3.0
```

Comment:
why we don't use the same csi-provisioner tag than in Centos example

Reply:
done